### DISCUSSION TITLE
Trying to read my multimeter

### DISCUSSION THREAD

**Discussion by i-g-g-y** [jomjol/AI-on-the-edge-device#2661] [Q&A]
The picture looks pretty good, but the recognition does not work at all?

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/10361323/c612cca8-7ede-4295-b0c6-89a80464456e)

**Comment by caco3** [comment]
this should work, others also use such LCD displays.
Try to rotate it a bit and make the ROIs have different Y positions to compensate for the italic font.
The reflection might also be an issue for the left digits.

And please provide a screenshot of your ROIs.

**Comment by i-g-g-y** [comment]
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/10361323/ad46da27-0d82-45cb-92f1-afdfcb970dcf)

**Reply by caco3** [reply]
I think your ROIs are too small, please have a look on https://jomjol.github.io/AI-on-the-edge-device-docs/ROI-Configuration/#how-to-setup-the-digit-rois-perfectly

**Reply by i-g-g-y** [reply]
I will give it a try. Right now the image is being aligned..

**Reply by i-g-g-y** [reply]
Better, but not there yet..
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/10361323/62c816f4-0625-4acc-84b9-907e220ee9e3)

**Reply by caco3** [reply]
Note that the values shown under "Recognized Digit ROIs (previous round)" are before the post processing algorithm!
Because of this I would expect you should get a correct 0.084 now.

See my gas meter where 2 digits are slightly before zero thus recognized as `9.9`.  The Post post processing will handle this correctly as `7803.470`:
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/1783586/0db8a206-738a-48ea-a8a6-d7fb570380a4)

**Reply by i-g-g-y** [reply]
Changed value:
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/10361323/9223e76d-20cc-4bf5-9373-60368d322c20)

But it reads 0704:
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/10361323/a441fdc7-26f6-4d1f-82aa-e98e18b1e39e)

**Reply by i-g-g-y** [reply]
Now recognized OK:
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/10361323/49aaf182-a4fd-4d3a-b10e-3dc4e2d941bc)

**Reply by i-g-g-y** [reply]
Still issues:

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/10361323/6d87cd19-e9d4-4ede-bdb5-c02e85de2bd2)

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/10361323/1f32f86e-4758-4551-9d79-35a6ccf19f1d)

**Reply by caco3** [reply]
"Rate to high". Du musst bei dir folgendes deaktivieren (expert settings):
- https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#PostProcessing-NUMBER.AllowNegativeRates muss deaktiviert werden
- https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-numbermaxratevalue muss auf etwas ganz grosses gesetzt werden

**Reply by caco3** [reply]
Zeig mal ein screenshot vom digits ROI setup. Irgendwie passen da die Abstände noch nicht.

**Reply by i-g-g-y** [reply]
![Uploading image.png…]()

**Reply by i-g-g-y** [reply]
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/10361323/ac2440e5-5a46-4b8c-b263-ebd15d80df05)

**Reply by i-g-g-y** [reply]
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/10361323/92132c40-a551-4f4f-a9d0-e0540d71c2a8)

**Reply by caco3** [reply]
Rotiere das Bild mal so fest, dass die senkrechten Balken wirklich senkrecht sind.

P.S. Wie verhinderst Du, dass dein Multimeter nicht automatisch ausschaltet?

**Reply by i-g-g-y** [reply]
This is the best I can do..
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/10361323/40194866-ede6-4eb9-b4e3-002a53ee0f96)

P.S: this unit has PC-link, which keeps it awake. But the PC SW does not work on my windows..

**Reply by i-g-g-y** [reply]
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/10361323/4f2b2113-ea73-4a27-ae1a-7073a1b767de)
